matisse/src/main/java/com/zhihu/matisse/internal/utils/MediaStoreCompat.java

```diff
@@ -93,6 +113,29 @@ public void dispatchCaptureIntent(Context context, int requestCode) {
         }
     }
 
+    private File createVideoFile() throws IOException {
+        // Create an image file name
+        String timeStamp =
+                new SimpleDateFormat("yyyyMMdd_HHmmss", Locale.getDefault()).format(new Date());
+        String videoFileName = String.format("MP4_%s.mp4", timeStamp);
+        File storageDir;
+        if (mCaptureStrategy.isPublic) {
+            storageDir = Environment.getExternalStoragePublicDirectory(
+                    Environment.DIRECTORY_PICTURES);
+        } else {
+            storageDir = mContext.get().getExternalFilesDir(Environment.DIRECTORY_MOVIES);
+        }
```

Comment:
On my MIUI8.3 and on my cyanogen11 camera app saves the files in DCIM. I think all media from the camera shold be saved in `Environment.DIRECTORY_DCIM`